flowmachine/tests/test_query_object_construction.py

```diff
@@ -36,22 +36,6 @@ def test_construct_query(diff_reporter):
                 "sampling": None,
             },
         },
-        {
-            "query_kind": "spatial_aggregate",
-            "locations": {
-                "query_kind": "daily_location",
-                "date": "2016-01-01",
-                "aggregation_unit": "admin3",
-                "method": "last",
-                "subscriber_subset": None,
-                "sampling": {
-                    "sampling_method": "system_rows",
-                    "size": 10,
-                    "fraction": None,
-                    "estimate_count": False,
-                },
-            },
-        },
```

Comment:
I wonder whether it's still worth having this test, but using a different sampling method?